[2025-01-09T12:33:57.637+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-09T12:33:57.736+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: movie_pipeline.parse_movies scheduled__2025-01-08T00:00:00+00:00 [queued]>
[2025-01-09T12:33:57.776+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: movie_pipeline.parse_movies scheduled__2025-01-08T00:00:00+00:00 [queued]>
[2025-01-09T12:33:57.780+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-09T12:33:57.825+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): parse_movies> on 2025-01-08 00:00:00+00:00
[2025-01-09T12:33:57.836+0000] {standard_task_runner.py:72} INFO - Started process 1343 to run task
[2025-01-09T12:33:57.843+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'movie_pipeline', 'parse_movies', 'scheduled__2025-01-08T00:00:00+00:00', '--job-id', '140', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpe8cjgpfx']
[2025-01-09T12:33:57.850+0000] {standard_task_runner.py:105} INFO - Job 140: Subtask parse_movies
[2025-01-09T12:33:58.082+0000] {task_command.py:467} INFO - Running <TaskInstance: movie_pipeline.parse_movies scheduled__2025-01-08T00:00:00+00:00 [running]> on host 78b847e880a5
[2025-01-09T12:33:58.771+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='movie_pipeline' AIRFLOW_CTX_TASK_ID='parse_movies' AIRFLOW_CTX_EXECUTION_DATE='2025-01-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-08T00:00:00+00:00'
[2025-01-09T12:33:58.774+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-09T12:33:59.430+0000] {parse.py:109} INFO - Parsing movie_id: tt13186482
[2025-01-09T12:34:00.253+0000] {parse.py:173} ERROR - time data 'Dec 20, 2024' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Dec 20, 2024' does not match format '%d-%b-%Y'
[2025-01-09T12:34:00.281+0000] {parse.py:175} WARNING - movie_id: tt13186482, release_date_id set to NONE
[2025-01-09T12:34:00.446+0000] {parse.py:109} INFO - Parsing movie_id: tt18259086
[2025-01-09T12:34:01.253+0000] {parse.py:173} ERROR - time data 'Dec 20, 2024' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Dec 20, 2024' does not match format '%d-%b-%Y'
[2025-01-09T12:34:01.255+0000] {parse.py:175} WARNING - movie_id: tt18259086, release_date_id set to NONE
[2025-01-09T12:34:01.344+0000] {parse.py:109} INFO - Parsing movie_id: tt5040012
[2025-01-09T12:34:02.486+0000] {parse.py:173} ERROR - time data 'Dec 25, 2024' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Dec 25, 2024' does not match format '%d-%b-%Y'
[2025-01-09T12:34:02.489+0000] {parse.py:175} WARNING - movie_id: tt5040012, release_date_id set to NONE
[2025-01-09T12:34:02.544+0000] {parse.py:109} INFO - Parsing movie_id: tt13622970
[2025-01-09T12:34:03.030+0000] {parse.py:173} ERROR - time data 'Nov 27, 2024' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Nov 27, 2024' does not match format '%d-%b-%Y'
[2025-01-09T12:34:03.032+0000] {parse.py:175} WARNING - movie_id: tt13622970, release_date_id set to NONE
[2025-01-09T12:34:03.123+0000] {parse.py:109} INFO - Parsing movie_id: tt11563598
[2025-01-09T12:34:03.600+0000] {parse.py:173} ERROR - time data 'Dec 25, 2024' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Dec 25, 2024' does not match format '%d-%b-%Y'
[2025-01-09T12:34:03.605+0000] {parse.py:175} WARNING - movie_id: tt11563598, release_date_id set to NONE
[2025-01-09T12:34:03.714+0000] {parse.py:109} INFO - Parsing movie_id: tt1262426
[2025-01-09T12:34:04.324+0000] {parse.py:173} ERROR - time data 'Nov 22, 2024' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Nov 22, 2024' does not match format '%d-%b-%Y'
[2025-01-09T12:34:04.326+0000] {parse.py:175} WARNING - movie_id: tt1262426, release_date_id set to NONE
[2025-01-09T12:34:04.392+0000] {parse.py:109} INFO - Parsing movie_id: tt9218128
[2025-01-09T12:34:05.248+0000] {parse.py:173} ERROR - time data 'Nov 22, 2024' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Nov 22, 2024' does not match format '%d-%b-%Y'
[2025-01-09T12:34:05.250+0000] {parse.py:175} WARNING - movie_id: tt9218128, release_date_id set to NONE
[2025-01-09T12:34:05.306+0000] {parse.py:109} INFO - Parsing movie_id: tt29137778
[2025-01-09T12:34:05.801+0000] {parse.py:173} ERROR - time data 'Dec 20, 2024' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Dec 20, 2024' does not match format '%d-%b-%Y'
[2025-01-09T12:34:05.805+0000] {parse.py:175} WARNING - movie_id: tt29137778, release_date_id set to NONE
[2025-01-09T12:34:05.909+0000] {parse.py:109} INFO - Parsing movie_id: tt6133444
[2025-01-09T12:34:06.338+0000] {parse.py:158} ERROR - 'NoneType' object has no attribute 'attrs'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 156, in movie_parser
    movie_data_dict['distributor_id']=child_span[1].find('a').attrs['href'].split('/')[-3]
AttributeError: 'NoneType' object has no attribute 'attrs'
[2025-01-09T12:34:06.341+0000] {parse.py:159} WARNING - movie_id: tt6133444, distributor_id set to NONE
[2025-01-09T12:34:06.343+0000] {parse.py:173} ERROR - time data 'Dec 25, 2024' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Dec 25, 2024' does not match format '%d-%b-%Y'
[2025-01-09T12:34:06.345+0000] {parse.py:175} WARNING - movie_id: tt6133444, release_date_id set to NONE
[2025-01-09T12:34:06.403+0000] {parse.py:109} INFO - Parsing movie_id: tt8790086
[2025-01-09T12:34:08.683+0000] {parse.py:173} ERROR - time data 'Dec 13, 2024' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Dec 13, 2024' does not match format '%d-%b-%Y'
[2025-01-09T12:34:08.686+0000] {parse.py:175} WARNING - movie_id: tt8790086, release_date_id set to NONE
[2025-01-09T12:34:08.941+0000] {parse.py:109} INFO - Parsing movie_id: tt23055660
[2025-01-09T12:34:10.220+0000] {parse.py:173} ERROR - time data 'Dec 13, 2024' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Dec 13, 2024' does not match format '%d-%b-%Y'
[2025-01-09T12:34:10.227+0000] {parse.py:175} WARNING - movie_id: tt23055660, release_date_id set to NONE
[2025-01-09T12:34:10.376+0000] {parse.py:109} INFO - Parsing movie_id: tt14824600
[2025-01-09T12:34:13.331+0000] {parse.py:173} ERROR - time data 'Dec 13, 2024' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Dec 13, 2024' does not match format '%d-%b-%Y'
[2025-01-09T12:34:13.334+0000] {parse.py:175} WARNING - movie_id: tt14824600, release_date_id set to NONE
[2025-01-09T12:34:13.394+0000] {parse.py:109} INFO - Parsing movie_id: tt2347285
[2025-01-09T12:34:13.904+0000] {parse.py:173} ERROR - time data 'Nov 8, 2024' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Nov 8, 2024' does not match format '%d-%b-%Y'
[2025-01-09T12:34:13.906+0000] {parse.py:175} WARNING - movie_id: tt2347285, release_date_id set to NONE
[2025-01-09T12:34:13.965+0000] {parse.py:109} INFO - Parsing movie_id: tt21823606
[2025-01-09T12:34:14.476+0000] {parse.py:173} ERROR - time data 'Nov 1, 2024' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Nov 1, 2024' does not match format '%d-%b-%Y'
[2025-01-09T12:34:14.479+0000] {parse.py:175} WARNING - movie_id: tt21823606, release_date_id set to NONE
[2025-01-09T12:34:14.568+0000] {parse.py:109} INFO - Parsing movie_id: tt20215234
[2025-01-09T12:34:15.922+0000] {parse.py:173} ERROR - time data 'Oct 25, 2024' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Oct 25, 2024' does not match format '%d-%b-%Y'
[2025-01-09T12:34:15.924+0000] {parse.py:175} WARNING - movie_id: tt20215234, release_date_id set to NONE
[2025-01-09T12:34:15.990+0000] {parse.py:109} INFO - Parsing movie_id: tt14948432
[2025-01-09T12:34:16.455+0000] {parse.py:173} ERROR - time data 'Nov 15, 2024' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Nov 15, 2024' does not match format '%d-%b-%Y'
[2025-01-09T12:34:16.457+0000] {parse.py:175} WARNING - movie_id: tt14948432, release_date_id set to NONE
[2025-01-09T12:34:16.522+0000] {parse.py:109} INFO - Parsing movie_id: tt28082769
[2025-01-09T12:34:17.097+0000] {parse.py:173} ERROR - time data 'Dec 13, 2024 (Jan 17, 2025)' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Dec 13, 2024 (Jan 17, 2025)' does not match format '%d-%b-%Y'
[2025-01-09T12:34:17.105+0000] {parse.py:175} WARNING - movie_id: tt28082769, release_date_id set to NONE
[2025-01-09T12:34:17.330+0000] {parse.py:109} INFO - Parsing movie_id: tt14260836
[2025-01-09T12:34:17.737+0000] {parse.py:158} ERROR - 'NoneType' object has no attribute 'attrs'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 156, in movie_parser
    movie_data_dict['distributor_id']=child_span[1].find('a').attrs['href'].split('/')[-3]
AttributeError: 'NoneType' object has no attribute 'attrs'
[2025-01-09T12:34:17.739+0000] {parse.py:159} WARNING - movie_id: tt14260836, distributor_id set to NONE
[2025-01-09T12:34:17.741+0000] {parse.py:173} ERROR - time data 'Dec 25, 2024 (Jan 10, 2025)' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Dec 25, 2024 (Jan 10, 2025)' does not match format '%d-%b-%Y'
[2025-01-09T12:34:17.742+0000] {parse.py:175} WARNING - movie_id: tt14260836, release_date_id set to NONE
[2025-01-09T12:34:17.801+0000] {parse.py:109} INFO - Parsing movie_id: tt29623480
[2025-01-09T12:34:18.646+0000] {parse.py:173} ERROR - time data 'Sep 27, 2024' does not match format '%d-%b-%Y'
Traceback (most recent call last):
  File "/opt/src/parse.py", line 167, in movie_parser
    movie_data_dict['release_date_id']=str(datetime.strptime(release_date,"%d-%b-%Y").date()).replace('-','')
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Sep 27, 2024' does not match format '%d-%b-%Y'
[2025-01-09T12:34:18.648+0000] {parse.py:175} WARNING - movie_id: tt29623480, release_date_id set to NONE
[2025-01-09T12:34:18.725+0000] {python.py:240} INFO - Done. Returned value was: run_20250106/movies_parsed_20250106.json
[2025-01-09T12:34:18.813+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-09T12:34:18.815+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=movie_pipeline, task_id=parse_movies, run_id=scheduled__2025-01-08T00:00:00+00:00, execution_date=20250108T000000, start_date=20250109T123357, end_date=20250109T123418
[2025-01-09T12:34:18.889+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-09T12:34:18.960+0000] {taskinstance.py:3895} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2025-01-09T12:34:18.966+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
